com.unity.render-pipelines.universal/Shaders/PostProcessing/UberPost.shader

```diff
@@ -9,6 +9,7 @@ Shader "Hidden/Universal Render Pipeline/UberPost"
         #pragma multi_compile_local_fragment _ _FILM_GRAIN
         #pragma multi_compile_local_fragment _ _DITHERING
         #pragma multi_compile_local_fragment _ _LINEAR_TO_SRGB_CONVERSION
+        #pragma multi_compile _ _USE_FAST_SRGB_LINEAR_CONVERSION
```

Comment:
UberPost has already many variants, a new multi-compile here will introduce a noticeable increase in build times. Can we avoid it?

Reply:
Made the property not changeable during runtime and added a shader preprocessor rule to avoid an increase in build times.